src/z_dateformat.erl

```diff
@@ -304,14 +305,26 @@ tag_to_value($w, Date, _, _Options) ->
 tag_to_value($W, {Y,M,D}, _, _Options) ->
    integer_to_list(year_weeknum(Y,M,D));
 
+% Absolute year with at least 4 digits, padded with zeroes (e.g. '0003')
+tag_to_value($x, {Y, _, _}, _, _Options) when Y >= 1000; Y =< -1000 ->
+    integer_to_list(erlang:abs(Y));
```

Comment:
This is wrong, as `$x` is used for ISO 8601 datetimes, where the year should be a negative integer. Why did you `abs` this?

Reply:
I added that, so that it was the same as the use of `abs` in the second clause of the previous commit:

```Erlang
% Year with at least 4 digits, padded with zeroes (e.g. '0003')
tag_to_value($x, {Y, _, _}, _, _Options) when Y >= 1000; Y =< -1000 ->
    integer_to_list(Y);
tag_to_value($x, {Y, _, _}, _, _Options) when Y < 0 ->
    lists:flatten(io_lib:format("~4..0B", [abs(Y)]));
tag_to_value($x, {Y, _, _}, _, _Options) ->
    lists:flatten(io_lib:format("~4..0B", [Y]));
```

Reply:
I more or less assumed (from the abs) that you want to use this in combination with BCE/CE notation, which needs an absolute year number.